client/dashboard/domains/domain-connection-setup/connection-mode-card.tsx

```diff
@@ -105,69 +105,72 @@ export default function ConnectionModeCard( {
 						<Text variant="muted">{ description }</Text>
 					</VStack>
 				</HStack>
-			</CardHeader>
-			{ isSelected && (
-				<CardBody>
-					<VStack spacing={ 4 }>
-						<VStack spacing={ 6 }>
-							{ mode === DomainConnectionSetupMode.SUGGESTED && ! hasEmailOrOtherServices && (
-								<Notice variant="info" title="No email or other services detected">
-									<Text>
-										{ __( 'You can safely connect your domain without affecting anything else.' ) }
-									</Text>
-								</Notice>
-							) }
-							{ mode === DomainConnectionSetupMode.SUGGESTED && hasEmailOrOtherServices && (
-								<Notice variant="warning" title="Email or other services detected">
-									<Text>
-										{ __(
-											'Warning: the services attached to your domain might be interrupted if you use this connection method'
-										) }
-									</Text>
-								</Notice>
-							) }
-							{ mode === DomainConnectionSetupMode.ADVANCED && hasEmailOrOtherServices && (
-								<Notice variant="info" title="Email or other services detected">
-									<Text>
-										{ __(
-											'To avoid disruption, this is the safest way to connect your domain name.'
-										) }
-									</Text>
-								</Notice>
-							) }
-							<Text>{ infoText }</Text>
+				{ isSelected && (
+					<>
+						<VStack spacing={ 4 } style={ { marginTop: '16px' } }>
```

Comment:
If we wrap the `HStack` element and the `isSelected` block with a `<VStack spacing={4} />` element, we can remove this top margin!

Reply:
Oh yes, thanks Luis!